features/sync/current_branch/feature_branch/merge_sync_strategy/no_push/enabled.feature

```diff
@@ -0,0 +1,45 @@
+Feature: no-push setting set to "true"
+
+  Background:
+    Given the current branch is a feature branch "feature"
+    And the commits
+      | BRANCH  | LOCATION | MESSAGE               |
+      | main    | local    | local main commit     |
+      |         | origin   | origin main commit    |
+      | feature | local    | local feature commit  |
+      |         | origin   | origin feature commit |
+    And Git Town setting "no-push" is "true"
```

Comment:
No need for adding a separate Git Town config setting since there is already the command-line switch.

```suggestion
```